content/en/docs/concepts/data-collection.md

```diff
@@ -41,11 +45,180 @@ The Collector is made up of the following components:
+[data formats](https://opentelemetry.io/docs/concepts/signals), and converts this data into 
+an internal data format recognized by the collector. Receivers can be push or pull based.
+
+You can set multiple protocols for a single receiver and make them listen to different ports by default. 
+The table below shows supported receiver formats for each signal type:
+
+| Signal Source        | Traces             | Metrics            | Logs               |
+| :---                 |    :----:          |  :---:             |               ---: |
+| Host Metrics         |                    | :heavy_check_mark: |                    |
+| Jaeger               | :heavy_check_mark: |                    |                    |
+| Kafka                | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| OpenCensus           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| OpenTelemetry (OTLP) | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Prometheus           | :heavy_check_mark: | :heavy_check_mark: |                    |
+| Zipkin               | :heavy_check_mark: |                    |                    |
+
+Learn how to configure receivers in the [configuration documentation](/docs/collector/configuration/#receivers).
+
+### Processors
+The job of the processor is to filter unwanted telemetry data and inject additional attributes to the data 
+before it is sent to the exporter. While receivers and exporters, the capabilities of processors differ immensely
+from one processor to the other. Processors are run on data between being received and being exported.
+While processors are optional, [some are
+recommended](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor#recommended-processors).
+
+The table below shows some currently supported processors, and the signals they
+possess:
+
+| Signal                 |       Traces        |      Metrics       |               Logs |
+|:-----------------------|:-------------------:|:------------------:|-------------------:|
+| Attributes             | :heavy_check_mark:  |                    | :heavy_check_mark: |
+| Batch                  | :heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: |
+| Filter                 |                     | :heavy_check_mark: |                    |
+| Memory Limiter         | :heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: |
+| Probabilistic Sampling | :heavy_check_mark:  |                    |                    |
+| Resource               | :heavy_check_mark:  | :heavy_check_mark: | :heavy_check_mark: |
+| Span                   | :heavy_check_mark:  |                    |                    |
+
+Processors are optional, although [some are 
+recommended](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor#recommended-processors). 
+
+Processors can be configured to export data into or consolidate data coming in from 
```

Comment:
Processor don't export data. Exporters export, processors process. I think it can be confusing to mix terminology like this.